xla/service/gpu/ir_emitter_triton_test.cc

```diff
@@ -2208,6 +2249,9 @@ ENTRY e {
 }
 
 TEST_F(TritonGemmLevel2Test, DoubleBroadcastOfScalarConstantIsHandled) {
+  if (SkipBF16Tests()) {
```

Comment:
Note: I think that many of these tests could be rewritten to not use `bf16`, thereby allowing the removal of this filter and allowing more coverage on rocm. Be that as it may, this is fine for now.